### DISCUSSION TITLE
ISO Download File Hashes Don't Match What is Posted

### DISCUSSION THREAD

**Discussion by Al-Green-COS** [Security-Onion-Solutions/securityonion#7770] [General] [answered]
Below are the respective hashes I got for the posted ISO. Please advise as to delta.

G:\ISO>Powershell Get-FileHash securityonion-2.3.110-20220407.iso -Algorithm MD5

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
MD5             A1206A818A1D9EDB1325972C465BD43C                                       G:\ISO\securityonion-2.3.110-20220407.iso

G:\ISO>Powershell Get-FileHash securityonion-2.3.110-20220407.iso -Algorithm SHA1

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA1            BFF00E6931727C171511CA07E94552FCD7C47124                               G:\ISO\securityonion-2.3.110-20220407.iso

G:\ISO>Powershell Get-FileHash securityonion-2.3.110-20220407.iso -Algorithm SHA256

Algorithm       Hash                                                                   Path
---------       ----                                                                   ----
SHA256          28A586661AB6151DF261EFB1F5973105793713D4230A921530EF28258EB59284       G:\ISO\securityonion-2.3.110-20220407.iso

**Comment by dougburks** [comment, marked as answer]
I just tested and I get the same hashes that are listed at https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md:
```
md5sum securityonion-2.3.110-20220407.iso
928d589709731efe9942ca134a6f4c6b  securityonion-2.3.110-20220407.iso

sha1sum securityonion-2.3.110-20220407.iso
ca588a684586cc0d5bde5e0e41c935ffb939b6c7  securityonion-2.3.110-20220407.iso

sha256sum securityonion-2.3.110-20220407.iso
cbf8743838af2c7323e629fb6b28d5dd00ae6658b0e29e4d0916411d2d526bd2  securityonion-2.3.110-20220407.iso
```

The most likely reason for your hashes being different is that your download of the ISO image was interrupted. Try your download again and, if you continue to have issues, consider downloading from a different network.

**Comment by Al-Green-COS** [comment]
Mr. Burks,

First, thank you for your prompt reply. Your assessment and recommendation were correct… there had been an issue with my initial download.

Upon completing a follow-up download, all hash values verified. In hindsight I should have suspected that an issue had occurred with the first download. Again, thank you for your prompt reply.

Respectfully,

Al Green

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
